src/coreclr/tools/Common/TypeSystem/Interop/IL/Marshaller.cs

```diff
@@ -1581,6 +1581,10 @@ protected override void EmitCleanupManaged(ILCodeStream codeStream)
 
     class AnsiStringMarshaller : Marshaller
     {
+        const int MAX_LOCAL_BUFFER_LENGTH = 256; // TODO: Is this accurate on all platforms?
```

Comment:
MAX_PATH on windows is `260` so will update this. assume same value is used on other platforms in the PAL.